File: ruby/fizz_buzz.rb
class FizzBuzz
    def self.turn(number)
        return "FizzBuzz" if number.modulo(3).zero? && number.modulo(5).zero?
        return "Fizz" if number.modulo(3).zero?
        return "Buzz" if number.modulo(5).zero?

        number
    end
end

File: php/FizzBuzz.php
<?php

class FizzBuzz
{
    public static function turn($number)
    {
        if (0 === $number % 3 && 0 === $number % 5) { return "FizzBuzz"; }
        if (0 === $number % 5) { return "Buzz"; }
        if (0 === $number % 3) { return "Fizz"; }

        return $number;
    }
}
